packaging/docker/run.sh

```diff
@@ -46,6 +46,16 @@ if [ -n "${PGID}" ]; then
   usermod -a -G "${PGID}" "${DOCKER_USR}" || echo >&2 "Could not add netdata user to group docker with ID ${PGID}"
 fi
 
+# Needed to read Proxmox VMs and (LXC) containers configuration files (name resolution + CPU and memory limits)
+HOST_WWW_DATA_GID=$(stat -c %g /host/etc/pve 2>/dev/null || true)
```

Comment:
And, in addition to that, I would much prefer if we would check if the file exists _before_ we try to get the GID. Relying on `stat` failing like this does mean less code, and theoretically faster code as well (common case of the file not being there is worse, average case is better), but it’s not immediately obvious what we’re doing here.

I had to reread this line after reading the rest of the changes to confirm that the logic is correct and we’re only running the code below if the file exists.